### ISSUE TITLE
AP-WC1E - local_reg failed: 403

### ISSUE THREAD

**Issue by JeffVH** [deiger/AirCon#183]
Hi,

Thanks for the integration!
I'm trying to configure the AirCon config for a Fujitsu General device. I am running the docker integration. 
The wireless module identifies as: "AP-WC1E".It gets identified by the integration and i get the lanip_key etc.

MQTT identifies the device and it is discoverable in Home Assistant, although always unavailable.
If I run the docker container in the foreground, I see a lot of messages stating:

aircon-0310_copy_config_1 exited with code 0
hisense_ac     | E0723 14:58:29.732  notifier.py:117] [KeepAlive] Sending local_reg failed: 403,
hisense_ac     | E0723 14:58:30.752  notifier.py:117] [KeepAlive] Sending local_reg failed: 403,
hisense_ac     | E0723 14:58:32.372  notifier.py:117] [KeepAlive] Sending local_reg failed: 403,
hisense_ac     | E0723 14:58:34.951  notifier.py:117] [KeepAlive] Sending local_reg failed: 403,

Am I missing something or did I forget a step?

What info do we need to find out what's happening?

Thank you in advance for all the help you can give.

Kind regards,
Jeff

**Comment by JeffVH** [comment]
First of all, sorry for creating an issue, I solved this myself.
Checking all the issues here, I found that local_reg.json only accepts posts from and to the same subnet.
I added an interface in that VLAN but still could not use the integration and received the 403 error code.

After some testing I used an Raspberry Pi I had laying around to install the integration in the same subnet.
Everything worked after that!

**Comment by JeffVH** [comment]
First of all, sorry for creating an issue, I solved this myself.
Checking all the issues here, I found that local_reg.json only accepts posts from and to the same subnet.
I added an interface in that VLAN but still could not use the integration and received the 403 error code.

After some testing I used an Raspberry Pi I had laying around to install the integration in the same subnet.
Everything worked after that!

**Comment by JeffVH** [comment]
First of all, sorry for creating an issue, I solved this myself.
Checking all the issues here, I found that local_reg.json only accepts posts from and to the same subnet.
I added an interface in that VLAN but still could not use the integration and received the 403 error code.

After some testing I used an Raspberry Pi I had laying around to install the integration in the same subnet.
Everything worked after that!
